man/coredumpctl.xml

```diff
@@ -248,6 +248,13 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--root=<replaceable>ROOT</replaceable></option></term>
+
+        <listitem><para>Use root directory <option>ROOT</option> when searching for journal files.
```

Comment:
"when searching for coredumps"

The journal files are an implementation detail, we shouldn't mention it in the docs